http/http.go

```diff
@@ -139,6 +145,19 @@ func (s *Server) CLICityHandler(w http.ResponseWriter, r *http.Request) *appErro
 	return nil
 }
 
+func (s *Server) CLICoordinatesHandler(w http.ResponseWriter, r *http.Request) *appError {
+	response, err := s.newResponse(r)
+	if err != nil {
+		return internalServerError(err)
+	}
+	var str string
+	str += FloatToString(response.LocationLatitude)
+	str += ","
+	str += FloatToString(response.LocationLongitude)
+	fmt.Fprintln(w, str)
```

Comment:
I think this can be simplified to `fmt.Fprintf(w, "%s, %s\n", formatCoordinate(response.Latitude, formatCoordinate(response.Longitude)`.